### DISCUSSION TITLE
Will it be possible to rank the questions from both sets?

### DISCUSSION THREAD

**Discussion by toxdes** [GATEOverflow/GATE-Marks-Evaluator#11] [Ideas]
Will it be possible to share 

1. `top-k-wrongly-attempted`
2. `top-k-not-answered`,
3.  `top-k-correctly-answered` 

for each set (subject + aptitude)? (where, `k= 5/10/15/20`)

I saw difficulty(`#easy`, `#normal`,`#difficult`) tags on GO, and thought they are calculated based on the above data. 

Tags would require some effort but a GO blog post would be easier? I don't know how it will help in general, but I personally find it very interesting.

**Comment by arjunsuresh** [comment]
Of course. This we had tried sometime back and is next in queue. What'll be my gate rank if GATE eliminated difficult questions? And difficulty determined by actual performance in GATE
